insuree_batch/apps.py

```diff
@@ -5,20 +5,33 @@
 DEFAULT_CFG = {
     "gql_query_batch_runs_perms": ["111102"],
     "gql_mutation_create_insuree_batch_perms": ["111101"],
+    "template_folder": "templates/insuree_batch",
```

Comment:
Is this folder under the assembly or the module folder. It has to be accessible even if the module is installed from PYPI so the best is to make sure the base folder is the assembly.

Reply:
Done